specifications/api/swagger.yaml

```diff
@@ -448,6 +473,7 @@ definitions:
       - depth
       - direction
       - inputs
+      - sources
```

Comment:
I am afraid that this isn't an acceptable change for our users :( 
We need to return resolved inputs when fetching the transaction history.